themes/night/theme.lua

```diff
@@ -0,0 +1,112 @@
+------------------------------------------
+-- Aesthetic Night awesomewm theme      --
+-- Created by https://github.com/rxyhn  --
+------------------------------------------
+
+local theme_assets = require("beautiful.theme_assets")
+local xresources = require("beautiful.xresources")
+local rnotification = require("ruled.notification")
+local dpi = xresources.apply_dpi
+
+local gfs = require("gears.filesystem")
+local themes_path = gfs.get_themes_dir()
```

Comment:
(This is also on other themes) I'm not sure why we don't do `themes_path = gfs.get_themes_dir() .. "night/` :shrug:

Reply:
as u wish sir, already fixed

Reply:
Nice !